src/coreclr/nativeaot/Runtime/startup.cpp

```diff
@@ -48,7 +48,7 @@ static bool DetectCPUFeatures();
 
 extern RhConfig * g_pRhConfig;
 
-#if defined(HOST_X86) || defined(HOST_AMD64) || defined(HOST_ARM64)
+#if defined(HOST_X86) || defined(HOST_AMD64) || defined(HOST_ARM64) || defined(HOST_LOONGARCH64)
```

Comment:
```suggestion
#if defined(HOST_X86) || defined(HOST_AMD64) || defined(HOST_ARM64)
```
This is needed only once LoongArch intrinsic support comes online